samples/examples/income_classifier/Makefile

```diff
@@ -6,22 +23,14 @@ infer.md: infer.ipynb
 		--to markdown \
 		--output infer.md
 
-train.md: train.ipynb
-	jupyter nbconvert \
-		train.ipynb \
-		--ClearOutputPreprocessor.enabled=False \
-		--to markdown \
-		--output train.md
-
-
 .PHONY: upload_artifacts
 upload_artifacts:
-	gsutil cp -R classifier gs://seldon-models/scv2/examples/income
-	gsutil cp -R explainer gs://seldon-models/scv2/examples/income
-	gsutil cp -R drift-detector gs://seldon-models/scv2/examples/income
-	gsutil cp -R outlier-detector gs://seldon-models/scv2/examples/income
-	gsutil cp -R preprocessor gs://seldon-models/scv2/examples/income
-	gsutil cp -R infer-data gs://seldon-models/scv2/examples/income
+	gsutil cp -R preprocessor gs://seldon-models/scv2/examples/${MLSERVER_FOLDER}/income
+	gsutil cp -R classifier gs://seldon-models/scv2/examples/${MLSERVER_FOLDER}/income
+	gsutil cp -R explainer gs://seldon-models/scv2/examples/${MLSERVER_FOLDER}/income
+	gsutil cp -R drift-detector gs://seldon-models/scv2/examples/${MLSERVER_FOLDER}/income
+	gsutil cp -R outlier-detector gs://seldon-models/scv2/examples/${MLSERVER_FOLDER}/income
+	gsutil cp -R infer-data gs://seldon-models/scv2/examples/${MLSERVER_FOLDER}/income
```

Comment:
how are we creating mlserver `model-settings.json`?

Reply:
They are fixed not generated

Reply:
ok